src/views/Orders.vue

```diff
@@ -246,6 +252,28 @@ export default defineComponent({
         });
       return deleteBatchAlert.present();
     },
+    async skipBatch (batch: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurrence based on the temporal expression.'),
+          buttons: [
+            {
+              text: this.$t("Don't skip"),
+              role: 'cancel',
+            },
+            {
+              text: this.$t('Skip'),
+              handler: async () => {
+                this.store.dispatch('job/skipJob', batch).then(() => {
+                  showToast(translate("This job has been skipped"))
```

Comment:
Improve the condition to display the toast only when we have success in response, currently, this message will always be displayed even if the API returns some error.

Reply:
Sure sir! Working on it.